src/scenes/ContactUs/index.tsx

```diff
@@ -38,10 +40,10 @@ export default function ContactUsView(): JSX.Element {
       title: strings.TITLE_REPORT_PROBLEM,
       description: strings.formatString(
         strings.DESCRIPTION_REPORT_PROBLEM,
-        <i>"{process.env.REACT_APP_TERRAWARE_FE_BUILD_VERSION || 'n/a'}"</i>
+        <i>"{useAppSelector(selectAppVersion) || 'n/a'}"</i>
```

Comment:
Thanks for this change, looks good. One suggestion is to move this out into a constant.